components/compliance-service/habitat/default.toml

```diff
@@ -22,4 +22,4 @@ manual_polling_interval = 1440
 [agent]
 workers = 10
 buffer_size = 1000
-remote_inspec_version = "REPLACE-FROM-PLAN.SH"
+remote_inspec_version = "4.7.24"
```

Comment:
i'll take this out of the pr ...also we need to figure out a way to not always run into this problem